pages/participantinfo.tsx

```diff
@@ -13,79 +13,143 @@ import {
                 <Text fontWeight="700" fontSize="36px">
                     Participant Information
                 </Text>
                 <Stack spacing={8}>
                     <Progress
-                        value={progressBar}
+                        value={getProgressBarValue(pageNum)}
                         size="sm"
                         colorScheme="blackAlpha"
                     />
-
-                    <HStack spacing="24px">
-                        <FormControl id="first-name">
-                            <FormLabel>First name</FormLabel>
-                            <Input placeholder="First name" />
-                        </FormControl>
-                        <FormControl id="last-name">
-                            <FormLabel>Last name</FormLabel>
-                            <Input placeholder="Last name" />
-                        </FormControl>
-                    </HStack>
-                    <FormControl id="date-of-birth">
-                        <FormLabel>Date Of Birth</FormLabel>
-                        <Input placeholder="Date Of Birth" />
-                    </FormControl>
-                    <FormControl id="street-address">
-                        <FormLabel>Street Address</FormLabel>
-                        <Input placeholder="Street Address" />
-                    </FormControl>
-                    <HStack spacing="24px">
-                        <FormControl id="postal-code">
-                            <FormLabel>Postal Code</FormLabel>
-                            <Input placeholder="Postal Code" />
-                        </FormControl>
-                        <FormControl id="city">
-                            <FormLabel>City</FormLabel>
-                            <Input placeholder="City" />
-                        </FormControl>
-                        <FormControl id="province">
-                            <FormLabel>Province</FormLabel>
-                            <Input placeholder="Province" />
-                        </FormControl>
-                    </HStack>
-                    <FormControl id="school">
-                        <FormLabel>School (if applicable)</FormLabel>
-                        <Input placeholder="School (if applicable)" />
-                    </FormControl>
-                    <FormControl id="grade">
-                        <FormLabel>Grade (if applicable)</FormLabel>
-                        <Input placeholder="Grade (if applicable)" />
-                    </FormControl>
-                    <Button
-                        backgroundColor="brand.500"
-                        color="brand.100"
-                        width="366px"
-                        height="44px"
-                        fontSize="14px"
-                        fontWeight="400"
-                        mt="20px"
-                    >
-                        Next
-                    </Button>
+                    {formPages[pageNum]}
```

Comment:
@sohak22 this is what I did to render the correct form page based on the page number. The form header also needs to change depending on what form page we are on (e.g. Parent/Guardian Information, Participant Emergency Form, etc.). 

Maybe as an exercise you could try implementing the pagination for the form headers using the same approach?